lib/taskana-core/src/main/java/pro/taskana/user/internal/UserServiceImpl.java

```diff
@@ -181,6 +185,9 @@ private void insertIntoDatabase(User userToCreate) throws UserAlreadyExistExcept
       if (userToCreate.getGroups() != null && !userToCreate.getGroups().isEmpty()) {
         userMapper.insertGroups(userToCreate);
       }
+      if (userToCreate.getPermissions() != null && !userToCreate.getPermissions().isEmpty()) {
+        userMapper.insertPermissions(userToCreate);
```

Comment:
Here, you always call the Oracle method. Please add a non oracle method to the mapper